templates/parachain/README.docify.md

```diff
@@ -240,6 +241,127 @@ npx @acala-network/chopsticks@latest --chain-spec <path/to/chain_spec.json>
 support it at this moment. It can still be used to test a runtime in a full setup where it is started alongside a
 relay chain network (see [Parachain Template node](#parachain-template-node) setup).
 
+### Deploy to Paseo TestNet & See Your First Block
```

Comment:
More `#` means a smaller header, so if you want this to be the only thing in the ToC, it should be bigger than `Steps to Deploy`. Basically, you'd need to use `##` and for the sub-headers, to add one more `#` for each.

Also, this part is not visible in the README.md on your branch. Please regenerate it with docify.